api/bookmarks/bookmarksModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../../data/db-config.js');
+
+const getbookmarks = () => {
+  return db('bookmarks');
+};
+
+const getByUserId = async (id) => {
+  return await db('bookmarks').where('id', id);
+};
+
+const addbookmark = async (bookmark) => {
+  await db('bookmarks').insert(bookmark);
```

Comment:
I'm pretty sure you can just return this line and that would return the newly created bookmark. heres some code where i did the sam thing: https://github.com/Build-Week-Potluck-Planner-4/back-end/blob/main/api/potlucks/modal.js